src/main/java/pl/fracz/mcr/source/NoSelectedLineException.java

```diff
@@ -0,0 +1,6 @@
+package pl.fracz.mcr.source;
+
+import pl.fracz.mcr.comment.CommentNotAddedException;
+
+public class NoSelectedLineException extends CommentNotAddedException {
```

Comment:
Niewłaściwe dziedziczenie - NoSelectedLineException nie jest szczególnym przypadkiem CommentNotAddedException. Może to wpłynąć negatywnie na logikę obsługi błędów.